app/views/layouts/html_content.html.erb

```diff
@@ -1,43 +1,45 @@
 <!DOCTYPE html>
 <html lang="<%= I18n.locale %>">
-  <head>
-    <% if @file_type == 'jupyter-notebook' %>
-      <%= stylesheet_link_tag 'notebook_common' %>
-      <% if @current_user&.theme == 'dark' %>
-        <%= stylesheet_link_tag 'notebook_dark' %>
-      <% else %>
-        <%= stylesheet_link_tag 'notebook_light' %>
-      <% end %>
-      <%= javascript_include_tag 'require', nonce: true %>
-      <%= javascript_tag nonce: true do %>
-        // Same as mathjax_config.js except startup.typeset is true
-        window.MathJax = {
-          startup: {
-            typeset: true
-          },
-          tex: {
-            // Allow inline single dollar sign notation
-            inlineMath: [
-              ["$", "$"],
-              ["\\(", "\\)"]
-            ],
-            processEnvironments: true,
-            processRefs: false
-          },
-          options: {
-            ignoreHtmlClass: "tex2jax_ignore",
-            processHtmlClass: "tex2jax_process"
-            },
-          svg: {
-            fontCache: "global"
-          }
-        };
-      <% end %>
-      <%= javascript_include_tag 'markus_notebook', nonce: true %>
-      <%= javascript_include_tag 'plotly', nonce: true %>
+<head>
+  <% if @file_type == 'jupyter-notebook' %>
+    <%= stylesheet_link_tag 'notebook_common' %>
+    <% if @current_user&.theme == 'dark' %>
```

Comment:
I think the choice of theme should actually apply to both the notebook and Rmd formats. You should test out your changes on dark mode as well (modify this under the account settings)